docs/Xamarin.Forms/Color.xml

```diff
@@ -1516,7 +1516,7 @@
         <ReturnType>Xamarin.Forms.Color</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Fucshia, the color that is represented by the RGB value #ff00ff. Deprecated. See Remarks.</summary>
+        <summary>Fuchsia, the color that is represented by the RGB value #ff00ff. Deprecated. See Remarks.</summary>
```

Comment:
I think this should be reverted, so that the spelling in the summary matches the API (which is misspelled and is now deprecated).